src/preprocessing/passes/rewrite.cpp

```diff
@@ -0,0 +1,41 @@
+/*********************                                                        */
+/*! \file rewrite.cpp
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Caleb Donovick
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2018 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ **/
+
+#include "preprocessing/passes/rewrite.h"
+
+#include "theory/theory.h"
```

Comment:
I think you don't need to include `theory/theory.h` but instead `theory/rewriter.h`.